guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -26,7 +26,39 @@ You cannot use ISS to synchronize content from {ProjectServer} to {SmartProxySer
 {SmartProxyServer} supports synchronization natively.
 For more information, see {PlanningDocURL}chap-Documentation-Architecture_Guide-Capsule_Server_Overview[{SmartProxyServer} Overview] in _Planning for {ProjectNameX}_.
 
-==== Air gapped
+=== Configuration Definitions
+There are different ways of using ISS to synchronize content between two {ProjectServer}s.
+
+.Connected
+In a connected scenario, two {ProjectServer}s are connected and are able to communicate with each other.
+
+ifndef::satellite[]
+image::Connected.png[title="The {Project} ISS Connected use case", alt="The {Project} ISS Connected use case"]
+endif::[]
+
+.Disconnected
+In a disconnected scenario, there is the following setup:
```

Comment:
I think this is supposed to be a list of items but it's being rendered in a paragraph with asterisks interspersed